<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePS</class>
 <widget class="QWidget" name="changePS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnChange">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>84</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>确定修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>84</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineOld">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 1px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNew">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 1px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>旧密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="labUser">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
